ngraph/core/include/ngraph/pass/low_latency.hpp

```diff
@@ -42,11 +43,50 @@ namespace ngraph
          * by step, the states will store between inferences.
          */
 
-        class NGRAPH_API LowLatency : public ngraph::pass::MatcherPass
+        class NGRAPH_DEPRECATED("Use LowLatency2 instead.") NGRAPH_API LowLatency
+            : public ngraph::pass::MatcherPass
         {
         public:
             NGRAPH_RTTI_DECLARATION;
             LowLatency();
         };
+
+        /**
+         * @brief The transformation finds all TensorIterator/Loop layers in the network,
+         * processes all back edges that describe a connection between Result and Parameter
+         * of the TensorIterator/Loop bodies,and inserts ReadValue and Assign layers at the
+         * input and output corresponding to this back edge.
+         * Supported platforms: CPU, GNA.
+         *
+         * The example below describes the changes made by the transformation
+         *  [] - TensorIterator body
+         *  () - new layer
+         *  BE - back-edge
+         *
+         *  before applying the transformation:
+         *  -> input1[BE_1 -> Parameter -> Layers ... -> Result  -> BE_1 ]output1->
+         *
+         *  after applying the transformation:
+         *  ->(ReadValue)-> input1[BE_1 ->Parameter->Layers ...->Result->BE_1]output1 ->(Assign)
+         *                                                                      \
+         *                                                                       ->...
+         * After applying the transformation, the resulting network can be inferred
+         * step by step, the states will store between inferences.
+         */
+        class NGRAPH_API LowLatency2 : public ngraph::pass::FunctionPass
```

Comment:
Is it possible to add new tests to nGraph unit tests for this transformation?

Reply:
already done` inference-engine/tests/functional/inference_engine/transformations/low_latency_v2_test.cpp `